xml/art_sle_ha_pmremote.xml

```diff
@@ -115,13 +144,14 @@
      </listitem>
     </varlistentry>
     <varlistentry>
-     <term>Remote Node</term>
+     <term>Physical Remote Node</term>
```

Comment:
That is a good idea! I'll fix "physical remote node" -> "remote node" everywhere in the doc.